cli/Valet/DevTools.php

```diff
@@ -132,6 +132,26 @@ public function vscode()
         }
     }
 
+    public function subl()
+    {
+        info('Opening Sublime Text');
+        $command = false;
+
+        if ($this->files->exists('/usr/local/bin/subl')) {
+            $command = '/usr/local/bin/subl';
+        }
```

Comment:
The [official Sublime documentation encourages](http://www.sublimetext.com/docs/3/osx_command_line.html) symlinking to `~/bin/subl`, so you might want to check that as well. (It is odd for them to assume that's in the PATH but what can ya do)

Reply:
Thanks for the feedback, it seems indeed legit to test the recommended path too. What do you think about testing the `subl` command with `which subl` to be more open to custom paths?

Reply:
I dig it 👍

Reply:
I wonder if it makes sense to pass in the `--new-window` argument as well, since this will be loading in the entire current Valet project

Reply:
I did some tests and it seems that folders are always opened in a new window, with or without the `--new-window` argument. I'll check with some coworkers using Sublime Text to confirm the behavior as I did not find anything in the `subl` command documentation.

Reply:
I updated the command to use `which subl` if you want to review it.